scripts/devops_tasks/test_run_samples.py

```diff
@@ -21,7 +21,12 @@
 root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", ".."))
 
 IGNORED_SAMPLES = {
-    "azure-eventgrid": ["__init__.py"]
+    "azure-eventgrid": [
+        "__init__.py",
```

Comment:
dumb question for my own learning: why here is a `__init__.py`, I don't see there is a `__init__.py` inside the sample folder?

Reply:
the __init__.py is from the azurefunctions appi was talking about in my previous comment
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventgrid/azure-eventgrid/samples/consume_samples/e2e_samples/functionsapp

Reply:
oh I see, then this is for EG only, make sense to me